Dockerfile

```
@@ -2,9 +2,32 @@ FROM ubuntu:24.04
+    xclip \
+    language-pack-ja \
+    build-essential
+
+# ホームディレクトリを英語名にする
+# RUN LANG=C xdg-user-dirs-gtk-update
+
+# Ubuntu Japanese
+RUN wget https://www.ubuntulinux.jp/ubuntu-jp-ppa-keyring.gpg -P /etc/apt/trusted.gpg.d/
+RUN wget https://www.ubuntulinux.jp/ubuntu-ja-archive-keyring.gpg -P /etc/apt/trusted.gpg.d/
+RUN wget https://www.ubuntulinux.jp/sources.list.d/jammy.list -O /etc/apt/sources.list.d/ubuntu-ja.list
+RUN sudo apt update && apt install -y ubuntu-defaults-ja
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)